### PR TITLE
DellSc Config deployment refactoring

### PR DESCRIPTION
**Pull request by rajinir**
Use the environment files(heat templates)
instead of extra config ( puppet directly)
This will help with upgrades

### CODE REVIEW

**Comment by gaelrehault** [comment]
build s failing, can you review for pep compliance please (particularly director.py)

**Comment by rajinir** [comment]
The pep8 errors were already existing. Fixed them anyways

**Comment by rajinir** [comment]
JT and Gael, please review now

File: src/deploy/osp_deployer/director.py [hunk 1]
```diff
@@ -1656,28 +1677,28 @@ def configure_tempest(self):
...
-                      + " -o -z '$(ls -A " + self.tempest_directory
-                      + " 2>/dev/null)' ]; then tempest init "
-                      + self.tempest_directory + ";fi;cd "
-                      + self.tempest_directory
-                      + ";discover-tempest-config --deployer-input "
-                      "~/tempest-deployer-input.conf --debug --create "
-                      "--network-id " + external_sub_guid
-                      + " object-storage-feature-enabled.discoverability "
-                      "False object-storage-feature-enabled.discoverable_apis"
+        cmd_route = ("source ~/" + setts.overcloud_name + "rc;" +
```
Comment (jtwill98) [inline src/deploy/osp_deployer/director.py]: I see you fixed this as the "+" was missing in the original.  Did the original work?

**Comment by jtwill98** [comment]
Thank you for the pep8 fixes.  I noticed you caught one issue fixed with the pep8 changes (see inline comment).  +1, waiting on Gael's review.

**Comment by jtwill98** [comment]
I didn't mean to close only comment.

File: src/deploy/osp_deployer/director.py (hunk 1 above)
Comment (rajinir) [reply src/deploy/osp_deployer/director.py]: yes

**Comment by jtwill98** [comment]
Merged as Gael is out and dev complete is tomorrow.    I reviewed it previously and want to provide code today for QA testing prior to code complete.  Additionally, Rajini reported that she deployed and tested the without issue.  Most changes were related to pep8 compliance.

**Pull request merged**